README.md

```diff
@@ -17,3 +17,11 @@ And an example conversation:
 
 <img src="https://user-images.githubusercontent.com/9945255/116060273-054fb080-a682-11eb-9fe4-d864305bf4d2.png" width="300" >
 
+## Setup using docker-compose
+1. See `niceday-api/` for getting the therapist user id and token. 
```

Comment:
This is still quite some manual work and depends in installing node modules correctly. I opened #45 to make this nicer.